readme.markdown

```diff
@@ -310,6 +311,17 @@ Outputs information about the package in the current working directory in
 an ini-compatible format. Useful for tools wishing to read metadata about
 Nimble packages who do not want to use the NimScript evaluator.
 
+### nimble sample
+
+The ``sample`` command copies from the packages samples directory to the
+working directory. (It does not creates new directory, and it overwrites 
+existing files)
+
+    myproject$ nimble sample [pkgname]
+    myproject$ nimble sample [pkgname] [samplename]
+    myproject$ nimble sample [pkgname] [pkgversion] [samplename]
```

Comment:
Package names and versions have a native syntax in Nimble: `pkg@0.1.0`. You should support this in this command instead of separating by a space.

Reply:
I'm also not sure how intuitive this command is, how does the user know what samples are available? Does the user look it up in the package's source code? If so, at that point it might just be easier to copy the file manually.

Perhaps a better approach would be (in general I would prefer the "example" nomenclature):

* ``nimble examples pkgname`` - displays a list of examples for `pkgname`
* ``nimble examples pkgname foobar`` - copies `foobar.nim` from `pkgname`'s examples to `./foobar.nim` (errors if `./foobar.nim` exists)
* ``nimble examples pkgname@0.1 foobar`` - copies `foobar.nim` from `pkgname`'s (at version 0.1) examples to `./foobar.nim` (errors if `./foobar.nim` exists)

What do you think?

Reply:
implemented, pushing in a minute